common/src/main/java/com/microsoft/identity/common/internal/broker/PackageHelper.java

```diff
@@ -124,7 +124,7 @@ private static String getSigningCertificateThumbprintForPackage(final PackageInf
             final Signature[] signatures = getSignatures(packageInfo);
             if (signatures != null && signatures.length > 0) {
                 final Signature signature = signatures[0];
-                MessageDigest md = MessageDigest.getInstance(useSha512 ? "SHA-512" : "SHA");
+                MessageDigest md = MessageDigest.getInstance(useSha512 ? "SHA-512" : "SHA"); // CodeQL [SM05136] MSAL still uses SHA-1 format in redirect url.
```

Comment:
SHA or SHA-1?

Reply:
"SHA" is a standard alias for SHA-1 in Android.